tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature

```diff
@@ -90,6 +90,7 @@ Feature: move files
     And user "Alice" has logged in using the webUI
     When the user tries to move file "lorem.txt" into folder "simple-folder (2)" using the webUI
     Then it should not be possible to paste files into the current folder using the webUI
```

Comment:
do these kind of steps do anything useful?

Reply:
It just checks if the confirm button is disabled and nothing useful. So removed.